dnf/cli/commands/__init__.py

```diff
@@ -199,12 +199,13 @@ def set_argparser(cls, parser):
 
     def configure(self):
         demands = self.cli.demands
-        demands.fresh_metadata = False
         demands.sack_activation = True
         if self.opts._packages_action:
             self.opts.packages_action = self.opts._packages_action
         if self.opts.packages_action != 'installed':
             demands.available_repos = True
+        else:
+            demands.fresh_metadata = False
```

Comment:
This change doesn't fix the behavior. Please remove ``else`` part or set ``demands.fresh_metadata = True``